managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2972,7 +2973,7 @@ public void asyncOffloadPrefix(Position pos, OffloadCallback callback, Object ct
     }
 
     private void offloadLoop(CompletableFuture<PositionImpl> promise, Queue<LedgerInfo> ledgersToOffload,
-            PositionImpl firstUnoffloaded, Optional<Throwable> firstError) {
+                             PositionImpl firstUnoffloaded, Optional<Throwable> firstError) {
```

Comment:
It's better to avoid changing the code format. Because we haven't a standard of that, everyone has their format. 
It is also difficult to review if there has too much code format changes.